java/code/src/com/suse/manager/matcher/MatcherJsonIO.java

```diff
@@ -158,12 +158,22 @@ public MatcherJsonIO() {
      * about systems on this Server
      */
     public List<SystemJson> getJsonSystems(boolean includeSelf, String arch, boolean selfMonitoringEnabled) {
+        List<String> vCoreCountedChannelFamilies = of("MICROOS-ARM64", "MICROOS-X86", "MICROOS-Z", "MICROOS-PPC")
```

Comment:
Nitpicking, but I'd use a `Set`, since the only operation you're doing is `contains()` and for that a set performs better.